servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/cancelPledge.js

```diff
@@ -1,10 +1,13 @@
 const logger = console
+const Promise = require('bluebird')
 const { Roles } = require('@orbiting/backend-modules-auth')
 const cancelMembership = require('./cancelMembership')
 
 const moment = require('moment')
 const { publishMonitor } = require('../../../../../lib/slack')
 
+const { evaluatePledge, updateMembershipPeriods } = require('../../../lib/Pledge/cancel')
+
 const {
   PARKING_PLEDGE_ID,
   PARKING_USER_ID
```

Comment:
all usages of `PARKING_PLEDGE_ID` and `PARKING_USER_ID` can be removed, no?

Reply:
There is a check to prevent cancellation of "parked" pledges which relay on these variables, and would like to stick to that check for a while longer.

Reply:
agreed!